Ackerman Function #1361 solution in python

pra17dod opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.

I wanted to add the solution of Ackermann Function

You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Description
I read each and every detail mentioned in the contributing.md

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This PR will add solution of issue #1361 on Ackermann Function in python in which I was assigned.

pra17dod committed:
Merge pull request #1 from TesseractCoding/master

Rebasing the forked repo

pra17dod committed:
Add ackermann_function.py solution

pra17dod committed:
Delete ackermann_function.py

pra17dod committed:
Add solution of issue Ackerman Function #1361

pra17dod committed:
Add Ackermann Function link

iamrajiv commented on Python/math/ackermann_function.py:
```diff
@@ -0,0 +1,36 @@
+def Ackermann_func(m, n): 
```
@pra17dod Add little bit description about Ackermann Function.

pra17dod replied:
Yaa sure

iamrajiv reviewed (changes requested):
@pra17dod Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

pra17dod committed:
Add requested changes by @imrajiv on PR #1382

Added description about Ackermann Function and formatted the code according to PEP 8 rules.

pra17dod commented:
Please review again the changes I made, @iamrajiv.

atarax665 commented on Python/math/ackermann_function.py:
```diff
@@ -0,0 +1,39 @@
+print("A(%u, %u) = %u" % (m, n, Ackermann_func(m, n)))
+
+# Time complexity: O( M * N )
+# Space complexity: O( M * N )
```
Add sample input/output in commented form.

pra17dod committed:
Add requested changes by @atarax665 on PR #1382

Added sample input and output.

pra17dod commented:
Please review again the changes I made, @atarax665 .

vatsalkesarwani12 reviewed (approved):
👍🏻
